x-pack/legacy/plugins/index_management/public/app/services/documentation.ts

```diff
@@ -160,6 +160,10 @@ class DocumentationService {
   public getWellKnownTextLink() {
     return 'http://docs.opengeospatial.org/is/12-063r5/12-063r5.html';
   }
+
+  public getRootLocaleLink() {
+    return 'https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#ROOT';
```

Comment:
I love that you added this link! Really helpful.

Reply:
Me too.  I worked on those docs.